service/arm_templates/main.json

```diff
@@ -468,7 +469,8 @@
           "osImageSKU": {"value": "[parameters('workersCustomConfig')[copyIndex()].osImage.sku]"},
           "osImageVersion": {"value": "[parameters('workersCustomConfig')[copyIndex()].osImage.version]"},
           "cloudConfigData": {"value": "[parameters('workerCloudConfigData')]"},
-          "loadBalancerBackendPoolIDs": {"value": "[createArray(reference('ingress_load_balancer_setup').outputs.loadBalancerBackendPoolID.value)]"}
+          "loadBalancerBackendPoolIDs": {"value": "[createArray(reference('ingress_load_balancer_setup').outputs.loadBalancerBackendPoolID.value)]"},
+          "assignContributorRole": {"value": "[bool('false')]"}
```

Comment:
If we also need to give the contributor role to the worker nodes, I'll just remove that condition as well as in the node_setup template and assign the role to all VMs

Reply:
We need MSI for all nodes in cluster.

Reply:
Se we can remove the flag as well.